bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+BRIDGE_PROPERTIES=$(cat <<-EOF
+#Bridge configuration
+bridge.id=${KAFKA_BRIDGE_ID}
```

Comment:
one more comment, from where the `KAFKA_BRIDGE_ID` comes? do you need to do additional work in the cluster operator for providing it from the `metadata.name` of the `KafkaBridge` resource?

Reply:
Yes, we will need to add this in the `KafkaBridgeCluster`.

Reply:
Open an issue there and reference this PR please.

Reply:
Addressed in https://github.com/strimzi/strimzi-kafka-operator/pull/1875